### PR TITLE
Add `IPlaceholder` Extensions Docs for Maui.Markup

### PR DESCRIPTION
**Pull request by brminnick**
This PR adds the Maui.Markup docs for the `IPlaceholder` extension methods added in this PR: https://github.com/CommunityToolkit/Maui.Markup/pull/44/files

### CODE REVIEW

**Comment by PRMerger18** [comment]
Microsoft authors of articles need to make updates through the private repository, so the updates can be staged and validated by current validation rules. After you review staging and fix all validation issues, you can add the sign-off comment to let the PR reviewers know the updates are ready to be merged. 

It's easy to access the private repo version of your articles in GitHub. Just add *-pr* to the URL, as follows: 

 &nbsp;Public repo URL: `https://github.com/MicrosoftDocs/{docs-repo}` 
 &nbsp;Private repo URL: `https://github.com/MicrosoftDocs/{docs-repo-pr}` 

 We have closed this pull request. Please resubmit the updates through the private repo. For more information, see the [contributor guide](https://docs.microsoft.com/contribute/).

**Comment by PRMerger15** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger15** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger9** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Comment by PRMerger17** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.

File: docs/maui/markup/extensions/placeholder-extensions.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Placeholder extensions - .NET MAUI Community Toolkit
+author: brminnick
+description: The Placeholder extensions provide a series of extension methods that support configuring `IPlaceholder` controls
```
Comment (jfversluis) [inline docs/maui/markup/extensions/placeholder-extensions.md:4]:
```suggestion
description: The Placeholder extensions provide a series of extension methods that support configuring IPlaceholder controls
```

You have no power here

**Review by jfversluis — APPROVED** [review]

**Comment by PRMerger17** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 File Change Percent | This PR contains file(s) with more than 20% file change.

**Pull request merged**